### PR TITLE
Master

### PR DESCRIPTION
**Pull request by timlinux**
### What does it fix?

Merge changes from develop to master

* Ticket: #
* Funded by: 
* Description:

### CODE REVIEW

File: safe/metadata/__init__.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 # pylint: disable=unused-import
 from safe.metadata.base_metadata import BaseMetadata
 from safe.metadata.generic_layer_metadata import GenericLayerMetadata
-from safe.metadata.exposure_summary_layer_metadata import (
-    ExposureSummaryLayerMetadata)
+from safe.metadata.output_layer_metadata import (
```
Comment (ismailsunni) [inline safe/metadata/__init__.py]: __[flake8]__
* [F401] 'safe.metadata.output_layer_metadata.OutputLayerMetadata' imported but unused

**Comment by Gustry** [comment]
Your PR looks wrong @timlinux, there is a lot of commit.
What command did you do ?

**Comment by Gustry** [comment]
It should be straight forward as it's a patch release

**Comment by timlinux** [comment]
```
git fetch upstream
git checkout master
git reset —hard upstream/master
git merge upstream/develop
git push origin master
```

**Pull request closed without merging**